modules/react-native-chat/Store/store.js

```diff
@@ -0,0 +1,21 @@
+// @ts-ignore
+import create from 'zustand';
+import { users } from './storage';
+const randIndex = Math.floor(Math.random() * users.length)
+export const uuid = users[randIndex]._id; // uuidv4()
+export const user = users[randIndex]
+export var ChannelType;
+(function (ChannelType) {
+  ChannelType[ChannelType["Direct"] = 0] = "Direct";
+  ChannelType[ChannelType["Group"] = 1] = "Group";
+})(ChannelType || (ChannelType = {}));
```

Comment:
What are you trying to accomplish here? I would avoid an IIFE (immediately invoked function expression) here.

Reply:
The above code was to create ChannelType(Group and Direct). I have rewritten the code.